modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
+You can use rule-based filtering to control the volume of packets cached in the eBPF flow table. For example, a filter can specify that only packets coming from port 100 should be recorded. Then all packets which do not match the filter are not cached and only the ones that match are cached.
+
+== How ingress and egress traffic filtering works
+CIDR notation allows for the efficient representation of IP address ranges by combining the base IP address with a prefix length. For both ingress and egress traffic, the source IP address is first used to match filter rules configured with CIDR notation. If the source IP address isn't configured, then the destination IP is used to match filter rules configured with CIDR notation. 
+
+After matching either the source IP or the destination IP CIDR, you can pinpoint specific endpoints using the `peerIP` to differentiate the destination IP address of the packet. Based on the provisioned action, the flow data is either cached in the eBPF flow table or not. 
+
+== Dashboard and metrics integrations
+When this option is enabled, the *Netobserv/Health* dashboard for *eBPF agent statistics* now has the *Filtered flows rate* view. Additionally, in *Observe* -> *Metrics* you can query `netobserv_agent_filtered_flows_total` to observe metrics with the reason in *FlowFilterAcceptCounter*, *FlowFilterNoMatchCounter* or *FlowFilterRecjectCounter*.
```

Comment:
does adding an image from console here of flowfilter metrics help ?

Reply:
At the beginning of NetObserv project, the team and I discussed whether or not to add screencaptures to the product documentation due to the heavy console pieces. Ultimately we decided that screencaptures would be a lot of maintenance overhead, which is why we have no screencaps in the docs today. So TL;DR, it may help but the concern is maintenance.